src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service;
```

Comment:
let's change the interfaces a little, because they are not completely clear right now
You should have 5 interfaces:
ReaderService - reading from a file
WriterService - writing to a file
ReportCreatorService - report creation
TransactionParserService - from list<String> makes list<Transaction>
TransactionsProcessingService - performs certain actions, depending on the input data
This will also solve the problem of dependencies of services on each other - they can be called one by one in the Main class